Mathlib/Data/DList/Basic.lean

```diff
@@ -50,4 +51,15 @@ theorem DList_lazy {α : Type _} {l : List α} : DList.lazy_ofList l = Std.DList
   rfl
 #align dlist_lazy Std.DList_lazy
 
+-- Porting note: port from lean3
+theorem DList.toList_ofList (l : List α) : DList.toList (DList.ofList l) = l := by
+  cases l; rfl; simp only [DList.toList, DList.ofList, List.cons_append, List.append_nil]
+
+-- Porting note: port from lean3
+theorem DList.ofList_toList (l : DList α) : DList.ofList (DList.toList l) = l := by
+   cases' l with app inv
+   simp only [ofList, toList, mk.injEq]
+   funext x
+   rw [(inv x)]
```

Comment:
Can we have some `#align` for these theorems

Reply:
Done